source/jquery.flot.navigate.js

```diff
@@ -115,7 +115,8 @@ can set the default in the options.
             active: false,
             cursor: "move",
             frameRate: 60,
-            mode: 'smart'
+            mode: 'smart',
+            allowRecenter: true
```

Comment:
Ugh. You're going to hate me a bit for this, but we need to revert this back to what you had before (a separate option for recenter), except the default needs to be true. Again, this is in service of providing new features without breaking existing functionality. The previous functionality allowed double-clicks to recenter _always_. It required no particular navigate option to be turned on. We should maintain this behavior.

Reply:
It is completely OK.